tests/ctst/steps/utils/utils.ts

```diff
@@ -374,6 +392,60 @@ async function addTransitionWorkflow(this: Zenko, location: string, enabled = tr
     }
 }
 
+function getReplicationLocationConfig(world: Zenko, location: string): {
+        storageClass: string;
+        destinationBucket: string;
+        locationType: string;
+    } {
+    switch (location) {
+    case 'awsbackendmismatch':
+        return {
+            storageClass: world.parameters.AwsBackendDestinationReplicationLocation,
+            destinationBucket: world.parameters.AwsReplicationBucketName,
+            locationType: world.parameters.AwsReplicationLocationType,
+        };
+    default:
+        throw new Error(`Unsupported replication location: '${location}'`);
+    }
+}
+
+async function putBucketReplication(
+    this: Zenko,
+    srcBucket: string,
+    replicationLocation: string
+) {
+    const { storageClass, destinationBucket } = getReplicationLocationConfig(this, replicationLocation);
```

Comment:
do we need to make things more generic? versus something like:
```suggestion
    const storageClass = world.parameters.AwsBackendDestinationReplicationLocation; 	
    const destinationBucket = world.parameters.AwsReplicationBucketName;
```

Or, if we really want to make things generic : give CTST access to the locationConfig secret (either retrieved and parsed whenever needed ; or mounted in the pod), and retrieve all needed informations by only specifying the name of the location in the tests...

Reply:
I want to keep the getReplicationLocationConfig() because with the other PR where I add another test for the replication failure case, I will use a different location, so I think this function will make more sense and become more useful with more tests

Reply:
I'm trying to get the locations list from inside the test now though, it will reduce the number of env variable we have to pass to the test

Reply:
done